src/common/crypto/mbedtls/cryptoprovider.cpp

```diff
@@ -228,6 +228,9 @@ static Error CreateClientCert(const mbedtls_x509_csr& csr, const mbedtls_pk_cont
         return AOS_ERROR_WRAP(err);
     }
 
+    notBefore.RightTrim("Z");
```

Comment:
According to the above mbedtls code snippet, it just add 'Z' at the end. So, it should work without `z` trimming. Shouldn't it?